buildSrc/src/main/groovy/airbyte-python.gradle

```diff
@@ -0,0 +1,98 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.CacheableTask
+import ru.vyarus.gradle.plugin.python.task.PythonTask;
+
+class AirbytePythonConfiguration {
+    String moduleDirectory
+}
+
+class AirbytePythonPlugin implements Plugin<Project> {
```

Comment:
Is this incremental?

Reply:
Not completely. The `use-python` configuration isn't cacheable. We could switch to running all of the commands ourselves and caching them in the future to have true incrementality. This actually might be preferable if we want to do something like run Python from Docker with certain tooling pre-installed.